### DISCUSSION TITLE
Removing files from LFS storage to make room for new files

### DISCUSSION THREAD

**Discussion by thipple505** [git-lfs/git-lfs#5726] [Q&A]
Hello,
I am pretty new to all of this and I had to download a git clone a repository that used the LFS for a larger file that's in that repo. I have been figuring out how to get that package running, however I have had to delete and clone the git repository a few times (I am new, lots of trial and error going on). I got an error saying that my account storage was full and I had to pay for more storage to git clone and use LFS any more. Is there any way for me to clear my remote storage (its all just copies of the same git clone)? I am on windows 11 and have primarily been using anaconda so far. Thanks for the help!

**Comment by chrisd8088** [comment]
Thanks for the question!  The answer depends on which Git hosting service you are using.

If you are using GitHub, their documentation [states](https://docs.github.com/en/repositories/working-with-files/managing-large-files/removing-files-from-git-large-file-storage#git-lfs-objects-in-your-repository) that deleting and recreating a repository will reset your Git LFS storage usage.  However, note that they also charge for Git LFS downloads and the bandwidth those consume.

Other Git hosting services, like GitLab, will have their own policies.  If you are still having trouble, I would recommend getting in touch with the Support team for whatever service you are using.  (This project is just for the open-source Git LFS client, so we can't help with billing issues for a particular service, other than providing generic advice.)
